tests/providers/sftp/sensors/test_sftp.py

```diff
@@ -97,11 +97,24 @@ def test_file_not_new_enough(self, sftp_hook_mock):
         sftp_hook_mock.return_value.get_mod_time.assert_called_once_with("/path/to/file/1970-01-01.txt")
         assert not output
 
+    @pytest.mark.parametrize(
+        "newer_than", (
+            datetime(2020, 1, 2),
+            datetime(2020, 1, 2, tzinfo=timezone.utc),
```

Comment:
Test failed due to `pendulum.timezone` is a function, I guess you want to use here `datetime.timezone.utc` here

So just import it and use here

```python
...
from datetime import timezone as stdlib_timezone
...
            datetime(2020, 1, 2, tzinfo=stdlib_timezone.utc),

```